src/damage.cpp

```diff
@@ -305,6 +305,17 @@ damage_type dt_by_name( const std::string &name )
     return iter->second;
 }
 
+std::string name_by_dt( const damage_type &dt ) {
```

Comment:
Declare such a variable within this function. As a static variable, it won't be deleted when the function goes out of scope, meaning it can be returned as reference.
For example, a function like this could exist:
```
const std::string &player::get_type_name()
{
    static const std::string type_name( "player" );
    return type_name;
}
```

Where unmodified, static strings are returned, they should be returned as references to save on copying.